src/modules/rppt_tensor_geometric_augmentations.cpp

```diff
@@ -1269,4 +1445,286 @@ return RPP_SUCCESS;
 #endif // backend
 }
 
+/******************** pixelate ********************/
+
+RppStatus rppt_pixelate_gpu(RppPtr_t srcPtr,
+                            RpptDescPtr srcDescPtr,
+                            RppPtr_t dstPtr,
+                            RpptDescPtr dstDescPtr,
+                            RpptROIPtr roiTensorPtrSrc,
+                            RpptRoiType roiType,
+                            rppHandle_t rppHandle)
+{
+#ifdef HIP_COMPILE
+    if ((srcDescPtr->dataType == RpptDataType::U8) && (dstDescPtr->dataType == RpptDataType::U8))
+    {
+        RpptInterpolationType interpolationType = RpptInterpolationType::BILINEAR;
+        RpptDesc interDesc;
+        interDesc = *srcDescPtr;
+        RpptDescPtr interDescPtr = &interDesc;
+        RpptImagePatchPtr internalDstImgSizes;
+        hipHostMalloc(&internalDstImgSizes, sizeof(RpptImagePatch) * rpp::deref(rppHandle).GetBatchSize());
```

Comment:
Similar comment as 
https://github.com/fiona-gladwin/rpp/pull/2/files#r1058870257

Please move the common code outside the if condition to avoid code replication